src/Compilers/Core/Portable/CodeGen/ItemTokenMap.cs

```diff
@@ -71,7 +68,7 @@ public IEnumerable<string> GetAllItems()
 
         //TODO: why is this is called twice during emit?
         //      should probably return ROA instead of IE and cache that in Module. (and no need to return count)
-        public IEnumerable<string> GetAllItemsAndCount(out int count)
+        public IEnumerable<T> GetAllItemsAndCount(out int count)
```

Comment:
This method turns out to have no references and can be deleted. GetAllItems above also returns an IEnumerable. Its only caller implements a CCI interface method that returns an IEnumerable, so there's presently little chance for anything to go wrong, but I can change it to return an ImmutableArray.